samples/drivers/jesd216/boards/nrf54l15pdk_nrf54l15_cpuapp.overlay

```diff
@@ -0,0 +1,15 @@
+/*
+ * Copyright (c) 2024 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+&mx25r64 {
+	status = "okay";
+};
+
+/ {
+	aliases {
+		jedec-spi-nor = &mx25r64;
```

Comment:
what are you trying to accomplish with this alias?

Reply:
@nordic-bami ping

Reply:
Hmm i have replied but it seems to not be visible:

DT_ macro in the test code uses such name

#if DT_HAS_COMPAT_STATUS_OKAY(jedec_spi_nor)

Reply:
You are mixing up concepts. The `mx25r64` node already has the compatible so you really only need to enable it. https://github.com/zephyrproject-rtos/zephyr/blob/main/boards/nordic/nrf54l15pdk/nrf54l15_cpuapp_common.dtsi#L137-L138

Please just drop these aliases, they are not correct/useless.

Reply:
Updated